.github/workflows/vercel.yml

```diff
@@ -17,13 +21,17 @@ jobs:
     environment: ${{ inputs.env_name }} # Environment rules defined on GH UI
     concurrency: ${{ inputs.env_name }} # Only one run per env at a time
 
+    strategy:
+      matrix:
+        project_suffix: ${{ inputs.app == 'ALL' && [COWSWAP, EXPLORER] || [input.app] }}
```

Comment:
After this we will need some process to add a new app, to remember to do all the steps
- add app in mono
- vercel setup
- edit here

Reply:
I will add a process to notion